### PR TITLE
Add prometheus step

### PR DESCRIPTION
**Pull request by stgleb**

### CODE REVIEW

File: templates/prometheus.sh.tpl [hunk 1]
```diff
@@ -0,0 +1,4 @@
+wget https://raw.githubusercontent.com/kubernetes/kubernetes/release-1.12/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/cluster-roles.yaml
+sudo kubectl create -f cluster-roles.yaml --validate=false
```
Comment (saheienko) [inline templates/prometheus.sh.tpl]: What if rbac is disabled?
```
pkg/model/kube.go:20:	RBACEnabled  bool      `json:"rbacEnabled"`
```

File: pkg/workflows/steps/prometheus/prometheus.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+)
+
+const StepName = "prometheus"
+
+type Step struct {
+	script *template.Template
+}
+
+func (s *Step) Rollback(context.Context, io.Writer, *steps.Config) error {
+	return nil
```
Comment (taaraora) [inline pkg/workflows/steps/prometheus/prometheus.go]: rollback was not supposed to be implemented?

File: templates/prometheus.sh.tpl (hunk 1 above)
Comment (taaraora) [inline templates/prometheus.sh.tpl]: ```sudo kubectl create -f cluster-roles.yaml``` looks like it is candidate for separate task?
Or may be it is better to use ```kubectl apply -f``` instead?

Comment (stgleb) [reply templates/prometheus.sh.tpl]: Well it won't work, helm chart requires cluster-admin role

Comment (stgleb) [reply templates/prometheus.sh.tpl]: Good point!

File: pkg/workflows/steps/prometheus/prometheus.go (hunk 2 above)
Comment (stgleb) [reply pkg/workflows/steps/prometheus/prometheus.go]: Rollback is not applicable for some sorts of steps. I don't think that it makes sense for this one.

Comment (stgleb) [reply pkg/workflows/steps/prometheus/prometheus.go]: Rollback is not applicable for some sorts of steps. I don't think that it makes sense for this one.

File: templates/prometheus.sh.tpl (hunk 1 above)
Comment (stgleb) [reply templates/prometheus.sh.tpl]: Btw, what if we enable RBAC by default without option to turn it off? Will it break some kinds of workload?

Comment (saheienko) [reply templates/prometheus.sh.tpl]: There is no authorization by default:
```
--authorization-mode stringSlice     Default: [AlwaysAllow]
--
  | Ordered  list of plug-ins to do authorization on secure port. Comma-delimited  list of: AlwaysAllow,AlwaysDeny,ABAC,Webhook,RBAC,Node.
```

to enable RBAC this should be added (apiserver has no config for it):
```
--authorization-mode=RBAC
```

**Pull request merged**